### PR TITLE
docs: add changelog 1.12.1

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/changelog/bytebase-1-12-1.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Improved onboarding flow.
+- Start a 14-day trial of the Enterprise Plan by default.
+- Improved Schema Editor's UX.
+- Improved Schema Diagram: filtering, navigation, focusing and export PNG files. [www.bytebase.com/docs/change-database/schema-diagram](/docs/change-database/schema-diagram)
+- Rename "Protected environment" to "Production environment". [www.bytebase.com/docs/administration/environment-policy/tier](/docs/administration/environment-policy/tier)
+- Improved schema viewing and editing experience in SQL Editor.
+- Add an icon to identify sensitive data columns in the query result of SQL editor.
+
+## 🐞 Notable Bug Fixes
+- Fixed: Migration history detail in MongoDB collection pages doesn't work correctly.
```
Comment (tianzhou) [inline content/changelog/bytebase-1-12-1.md:27]:
```suggestion
- Fixed: Change history detail in MongoDB collection pages doesn't work correctly.
```

File: content/changelog/bytebase-1-12-1.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## 🎠 Community
+- Thanks to [@gandergo](https://github.com/gandergo) for fix: allow postgres user name with hyphen [\#4236](https://github.com/bytebase/bytebase/pull/4236)
+- Thanks to [@alexey-milovidov](https://github.com/alexey-milovidov) for PR: Change ClickHouse Docker repository. [\#4648](https://github.com/bytebase/bytebase/pull/4648)
+
+## 📰 Fresh off the press
+- We introduced the MongoDB support in the last release. This is a milestone for us as it's also the first NoSQL database supported by Bytebase, [learn more](/blog/introducing-mongodb-support-in-bytebase)!
+
+- We officially announced [Terraform Bytebase Provider](/blog/introducing-terraform-bytebase-provider), which will allow you to codify the database configuration and consolidate your workflow.
```
Comment (tianzhou) [inline content/changelog/bytebase-1-12-1.md:36]: Also need to include partnership with GitHub

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/changelog/bytebase-1-12-1.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## 🚀 New Features
+- Support OAuth 2.0 for Single Sign-On (SSO) [www.bytebase.com/docs/administration/sso/oauth2](/docs/administration/sso/oauth2).
+- Support change and query for Spanner [www.bytebase.com/docs/introduction/supported-databases](/docs/introduction/supported-databases).
+- Support watermark [www.bytebase.com/docs/administration/watermark](/docs/administration/watermark).
+- Terraform Bytebase provider supports MySQL role CRUD.
+- Mask sensitive fields for PostgreSQL.
+
+## 🎄 Enhancements
+- For projects with GitOps workflow enabled, schema and data changes are also supported through UI.
```
Comment (Candybase) [inline content/changelog/bytebase-1-12-1.md:17]:
```suggestion
- For projects with GitOps workflow enabled, schema and data changes are also supported through UI.
- Supports executing statements in PostgreSQL that cannot be executed in a transaction, such as CREATE/DROP INDEX CONCURRENTLY.
```

**Review by Candybase — CHANGES_REQUESTED** [review]

**Review by tianzhou — APPROVED** [review]
LGTM

**Review by Candybase — APPROVED** [review]
LGTM

**Pull request merged**